### PR TITLE
Added python script to generate labelme json, README modified

### PR DESCRIPTION
**Pull request by ivrolan**
The script takes 2 arguments, the path to a folder and the name of the label to use in every json. The folder must contain:

- `masks` folder containing `.pbm` files (it can contain more than the masks that will be used)
- `.jpg` files to classify in a labelme json

It will generate as `.json` files as `.jpg` in the folder. The masks in the masks folder must be named the same as its corresponding images and followed by `_mask.pbm`.
> If we have `pepito.jpg` in the folder, the masks folder must contain a `pepito_mask.pbm` file. 

For example:

```
$ python3 create_labelme_json.py path_to_folder 'label to use'
```

Signed-off-by: ivrolan <ivanlpzbrc01@gmail.com>

### CODE REVIEW

File: create_labelme_json.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            y = float(coords[1])
+            points_list.append([x, y])
+    #print(points_list)
+    return points_list
+
+def main():
+    
+    if len(sys.argv) != 3:
+        print("2 parameters needed. [path] [label_name]")
+        return
```
Comment (fgonzalezr1998) [inline create_labelme_json.py]: Please, add a failure exit

File: create_labelme_json.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            
+            
+            f= open(img + ".json","w")
+
+            f.write(str(json.dumps(labelme_json, indent=4)))
+
+            print(img, "done")
+    
+if __name__ == "__main__":
+    main()
```
Comment (fgonzalezr1998) [inline create_labelme_json.py]: Successfully exit

File: create_labelme_json.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    return points_list
+
+def main():
+    
+    if len(sys.argv) != 3:
+        print("2 parameters needed. [path] [label_name]")
+        return
+
+    img_type = ".jpg"
+    os.chdir(sys.argv[1])
```
Comment (fgonzalezr1998) [inline create_labelme_json.py]: What happen if target folder doesn't exist?

File: create_labelme_json.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+import sys
+import json
+import numpy as np
+import cv2
+import base64
+import labelme
+
+def getJsonTemplate():
+
+    labelme_file_template = {
```
Comment (fgonzalezr1998) [inline create_labelme_json.py:11]: This should be a constant value

**Comment by ivrolan** [comment]
Thanks for the reviews!
Also I realize about a bug in the label name of the generated json, it should be fixed now. I'll fix the categories with this error ASAP. My apologies

**Pull request merged**